File: generateclasses/Matierestyle.java
package entite;

public class Matierestyle{
    int idmatierestyle; 
    int idmatiere; 
    int idstyle; 

    public Matierestyle(){ }
    
    public int getIdmatierestyle(){
        return this.idmatierestyle;
    }
    public void setIdmatierestyle(int idmatierestyle){
        this.idmatierestyle=idmatierestyle;
    }
    public int getIdmatiere(){
        return this.idmatiere;
    }
    public void setIdmatiere(int idmatiere){
        this.idmatiere=idmatiere;
    }
    public int getIdstyle(){
        return this.idstyle;
    }
    public void setIdstyle(int idstyle){
        this.idstyle=idstyle;
    }

}

File: generateclasses/Matierestyle.cs
namespace entite;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

[Table("matierestyle")]
public class Matierestyle : Motherobj<Matierestyle>  {
    [Key]
    [Column("idmatierestyle")]
    public int idmatierestyle {get; set;} 
    [Column("idmatiere")]
    public int idmatiere {get; set;} 
    [Column("idstyle")]
    public int idstyle {get; set;} 

    public Matierestyle(){ }
    

}
